pkg/http/request_log_test.go

```diff
@@ -243,3 +237,67 @@ func TestSetTemplate(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkRequestLogHandlerNoTemplate(b *testing.B) {
+	handler, err := NewRequestLogHandler(baseHandler, ioutil.Discard, "", defaultInputGetter, false)
+	if err != nil {
+		b.Fatalf("Failed to create handler: %v", err)
+	}
+	request := func() *http.Request {
```

Comment:
not sure this lambda is worth the lines it occupies.
Same with `test`.

Reply:
True.